include/event_manager.h

```diff
@@ -167,6 +167,9 @@ struct event_type {
 	 * subscribers. */
 	const struct event_subscriber	*subs_stop[SUBS_PRIO_COUNT];
 
+	/** Display event by default */
+	const bool init_log_enable;
```

Comment:
const is not needed as variables of this type go to flash (they are const)

Reply:
(note that log_event pointer is not const for example...)

Reply:
removed const

Reply:
not removed

Reply:
ok, fixed